org.eclipse.scanning.example/src/org/eclipse/scanning/example/malcolm/DummyMalcolmModel.java

```diff
@@ -60,14 +58,6 @@ public DummyMalcolmModel() {
 	public void setDummyDetectorModels(List<DummyMalcolmControlledDetectorModel> dummyDetectorModels) {
 		this.dummyDetectorModels = dummyDetectorModels;
 	}
-	
-	public long getTimeout() {
-		return timeout;
-	}
-
-	public void setTimeout(long timeout) {
-		this.timeout = timeout;
-	}
 
 	public List<String> getPositionerNames() {
 		if (positionerNames == null) return Collections.emptyList();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') At most one statement is allowed per line, but 2 statements were found on this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00122)